backend/app/tests/ui/test_cops_modal.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+
+from pages.home import Home
+
+
+@pytest.mark.smoke
+@pytest.mark.ui
+@pytest.mark.nondestructive
+def test_modal_remains_open_with_data_fields_empty(selenium, base_url):
+    # GIVEN: Selenium driver and the base url
+
+    # WHEN: The Home page is fully loaded
+    home = Home(selenium, base_url).open()
+    # AND: The create new pdf button is clicked
+    modal = home.click_create_new_pdf_button()
+
+    # AND: Create button is clicked when data fields are empty
+    modal.click_create_button()
+
+    # THEN: The create new pdf modal stays open
+    assert home.create_pdf_modal_is_open
+
+
+@pytest.mark.smoke
+@pytest.mark.ui
+@pytest.mark.nondestructive
+@pytest.mark.parametrize("colid", ["col26069"])
```

Comment:
these can be combined into one parametrize. 
Example: 
https://github.com/openstax/cnx-automation/blob/master/tests/neb/ui/test_get.py#L79

Documentation
https://docs.pytest.org/en/latest/parametrize.html#pytest-mark-parametrize